0110/streams/index.html

```diff
@@ -82,26 +82,8 @@ <h3 class="streams_intro">The easiest way to write mission-critical real-time ap
         <h3 class="stream__text">Streams API use cases</h3>
          <div class="customers__grid">
            <div class="customer__grid">
-             <div class="customer__item  streams_logo_grid streams__line__grid">
-               <a href="https://linecorp.com/" target="_blank" class="grid__logo__link">
-                 <span class="grid__item__logo" style="background-image: url('/images/powered-by/line.svg');width:9rem"></span>
-               </a>
-                 <p class="grid__item__customer__description extra__space"><a href="https://engineering.linecorp.com/en/blog/detail/80" target="_blank">LINE uses Apache Kafka</a> as a central datahub for our services to communicate to one another. Hundreds of billions of messages are produced daily and are used to execute various business logic, threat detection, search indexing and data analysis. LINE leverages Kafka Streams to reliably transform and filter topics enabling sub topics consumers can efficiently consume, meanwhile retaining easy maintainability thanks to its sophisticated yet minimal code base.</p>
-             </div>
-           </div>
-           <div class="customer__grid">
-             <div class="customer__item  streams_logo_grid streams__zalando__grid">
-               <a href="http://www.zalando.com" target="_blank" class="grid__logo__link">
-                 <span class="grid__item__logo" style="background-image: url('/images/powered-by/zalando.jpg');"></span>
-               </a>
-               <p class="grid__item__customer__description extra__space">As the leading online fashion retailer in Europe, Zalando uses Kafka as an ESB (Enterprise Service Bus), which helps us in transitioning from a monolithic to a micro services architecture. Using Kafka for processing
-                 <a href="https://kafka-summit.org/sessions/using-kstreams-ktables-calculate-real-time-domain-rankings/" target='blank'> event streams</a> enables our technical team to do near-real time business intelligence.
-               </p>
-             </div>
-           </div>
-           <div class="customer__grid">
              <div class="customer__item streams_logo_grid streams__ny__grid">
-               <a href="https://www.nytimes.com" target="_blank" class="grid__logo__link">
+               <a href="https://www.confluent.io/blog/publishing-apache-kafka-new-york-times/" target="_blank" class="grid__logo__link">
```

Comment:
Maybe better to use this url instead as it is from the original nyt website?

https://open.nytimes.com/publishing-with-apache-kafka-at-the-new-york-times-7f0e3b7d2077